receiver/kubeletstatsreceiver/metadata.yaml

```
@@ -13,6 +13,14 @@ resource_attributes:
     description: "The name of the Node"
     enabled: true
     type: string
+  k8s.node.labels:
+    description: "The labels of the Node"
+    enabled: true
```

```suggestion
    enabled: false
```

New resource attributes should not be enabled by default, need a breaking change